### PR TITLE
Newpizzacutter

### PR DESCRIPTION
**Pull request by Mensla**
# About the pull request

Adds two new pizza cutter sprites, one with a plastic handle and one with a wooden handle. Adds item sprites and hand-holding sprites.

No more comically large pizza cutter as they're now appropriately sized sprites in your hand.

# Explain why it's good for the game

Allows for people to use a normal and non-memey pizza cutter. 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

image
image

</details>

# Changelog

:cl:
add: Added new pizza cutter sprites.
/:cl:

### CODE REVIEW

**Comment by Mensla** [comment]
Sorry for the unspaced PR name

**Review by fira — CHANGES_REQUESTED** [review]
make one subtype of another, that'll be easier to use and more compact
bonus points if you also integrate the old cutter into that
also you could have "wood" matter in the wooden one but that's a meme 👌

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 1]
```diff
@@ -135,6 +135,46 @@
...
+	flags_atom = FPRINT|CONDUCT
+	sharp = IS_SHARP_ITEM_ACCURATE
+	edge = 1
+	force = 10
+	w_class = SIZE_MEDIUM
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	throwforce = 6
+	throw_speed = SPEED_VERY_FAST
+	throw_range = 6
+	matter = list("metal" = 12000)
```
Comment (Birdtalon) [inline code/game/objects/items/tools/kitchen_tools.dm:154]: 12000 metal isn't that quite a lot?

Comment (Mensla) [reply code/game/objects/items/tools/kitchen_tools.dm:154]: Same as the kitchen knives

Comment (Birdtalon) [reply code/game/objects/items/tools/kitchen_tools.dm:154]: Damn must be quadrillion folded steel.

**Comment by Mensla** [comment]
I created subtypes (?) and I think they're done correctly. 

Moved the meme pizza cutter under kitchen tools for organization.

**Comment by Birdtalon** [comment]
> I created subtypes and I think they're done correctly (?).
> 
> Moved the meme pizza cutter under kitchen tools for organization.
> 
> I have zero idea why the compile maps is failing...

maps/map_files/FOP_v3_Sciannex/sprinkles/30.pizzatime.dmm:308:error: undefined type: /obj/item/weapon/pizza_cutter
maps/map_files/FOP_v3_Sciannex/sprinkles/30.pizzatime.dmm:312:error: unknown type

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 2]
```diff
@@ -135,6 +135,44 @@
...
+
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/*
+ * Wood Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter/wood
+	icon_state = "woodpizzacutter"
+	desc = "A circular blade using for cutting pizzas. This one has an authetnic wooden handle."
```
Comment (BadAtThisGame302) [inline code/game/objects/items/tools/kitchen_tools.dm:163]: authentic*

File: code/game/objects/items/tools/kitchen_tools.dm [hunk 3]
```diff
@@ -135,6 +135,44 @@
 
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
+/*
+ * Plastic Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter
+	name = "pizza cutter"
+	icon_state = "plasticpizzacutter"
+	desc = "A circular blade using for cutting pizzas. This one has a cheap plastic handle."
```
Comment (BadAtThisGame302) [inline code/game/objects/items/tools/kitchen_tools.dm:144]: I also think used is better than using here, using doesn't make much sense grammar wise.

File: code/game/objects/items/tools/kitchen_tools.dm (hunk 2 above)
Comment (BadAtThisGame302) [reply code/game/objects/items/tools/kitchen_tools.dm:163]: Also the used/using issue as I said above.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Review by nauticall — APPROVED** [review]

**Comment by Birdtalon** [comment]
Resolve conflicts please this is ready to merge otherwise.

**Pull request closed without merging**